apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -26,8 +26,22 @@ function is_core_fse_active() {
 		return false;
 	}
 
-	// Now we just check for our own option.
-	return (bool) get_option( ACTIVATE_FSE_OPTION_NAME );
+	// We identify universal themes by assuming that they will all use blockbase as the
+	// default template.
+	$universal_theme_template = defined( 'IS_ATOMIC' ) && IS_ATOMIC
```

Comment:
I don't like relying on a check for the Atomic environment here--there's nothing specific about Atomic sites that changes theme behavior.

Falling back to the trick we were using before with the `basename` function to get the theme slug without the `pub/` or `premium/` prefix would be better.

Alternatively, we could check `IS_WPCOM`, but we'll need to account for both `pub` and `premium` themes (Videomaker will be a premium theme on wpcom with Blockbase as its parent, for example)

Reply:
Good suggestion! I went ahead and used `basename` to make the function environment agnostic.